backend/server/routers/followups.py

```diff
@@ -0,0 +1,86 @@
+from typing import Dict
+from fastapi import APIRouter
+from server.database import coursesCOL
+import json 
+
+router = APIRouter(
+    prefix="/followups",
+    tags=["followups"],
+)
+
+def get_next_term(term: str) -> str:
+    if term[0] == 'T':
+        if int(term[-1]) < 3:
+            return 'T' + str(int(term[-1]) + 1)
+        return 'S'
+    return 'T1'
+
+@router.post(
+    "/getFollowups/{origin_course}/{origin_term}",
+    responses={
+        200: {
+            "description": "Returns a list of the most popular followup courses",
+            "content": {
+                "application/json": {
+                    "example": {
+                        "originCourse": "COMP1511",
+                        "originTerm": "T2",
+                        "followups": {
+                            "COMP1521": {
+                                "T3": 339,
+                            },
+                            "COMP1531": {
+                                "T3": 200,
+                            },
+                            "COMP2521": {
+                                "T3": 120,
+                            },
+                            "COMP1511": {
+                                "T3": 45,
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+)
+def get_followups(origin_course: str, origin_term: str) -> Dict[str, str]:
```

Comment:
mypy is angry at you because of this line - you set the return type to be `Dict[str, str]`, but `followups`'s value is actually a dictionary, not a string.

Reply:
Whoops ok I changed the return but forgot to change the declaration, will fix this. (and get mypy)